okhttp-sse/src/main/java/okhttp3/package-info.java

```diff
@@ -0,0 +1,3 @@
+/** Support for server-sent events. */
+@javax.annotation.ParametersAreNonnullByDefault
+package okhttp3;
```

Comment:
I think the presence of this will break building on Android because it creates duplicate classes in the same package. Perhaps we should keep this stuff in `okhttp.sse` while developing?

Reply:
Also important for java 9 modules